src/schemas/uncle-block.json

```diff
@@ -0,0 +1,107 @@
+{
+	"UncleBlock": {

```

Comment:
Can you reuse the existing `Block` definition?

Reply:
You are probably right. I will have to edit it a little. I am looking into it now.

Reply:
After fixing the Block schema I don't think I can reuse it for the uncles.

Reply:
What was the issue? That `transactions` isn't a key on uncle blocks?